lib/storext/class_methods.rb

```diff
@@ -23,7 +23,7 @@ def store_attribute(column, attr, type=nil, opts={})
       track_store_attribute(column, attr, type, opts)
       storext_check_attr_validity(attr, type, opts)
       storext_define_accessor(column, attr)
-      store_accessor column, *storext_attrs_for(column)
+      store_accessor(column, attr)
```

Comment:
Yes, but that bug interacts with `storext-override`.  I attempted to implement similar behavior but could not reproduce it here.  I tried `def`ing an accessor, but it did not get overridden by the original code.

A separate issue is that the method `store_attribute` accepts a single attribute.  It seems incorrect to pass all existing attributes to `store_accessor` when only one needs to be added.